packages/asc-ui/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import { ChevronDown } from '@datapunt/asc-assets'
+import AccordionStyle, {
+  AccordionButton,
+  AccordionButtonContent,
+  AccordionContent,
+  Props as StyleProps,
+} from './AccordionStyle'
+
+type Props = {
+  id: string
+  title: string
+  onToggle?: Function
+} & StyleProps
+
+const Accordion: React.FC<Props & React.HTMLAttributes<HTMLDivElement>> = ({
+  children,
+  title,
+  id,
+  open: openProp,
+  onToggle,
+  ...otherProps
+}) => {
+  const [open, setOpen] = useState(openProp || false)
+
+  useEffect(() => {
+    if (typeof openProp !== 'undefined' && openProp !== open) {
+      setOpen(openProp)
+    }
+  }, [openProp])
+
+  const handleClick = useCallback(() => {
+    const newOpenState = !open
+    if (onToggle) {
+      onToggle(newOpenState)
+    }
+    setOpen(newOpenState)
+  }, [open])
+
+  return (
+    <AccordionStyle
+      {...otherProps}
+      open={open}
+      aria-expanded={open}
```

Comment:
Should't the aria-expanded be set on the button?

Reply:
ah yes apparently it should